fecgov/fec-cms
# Commit missing migration 0136 to develop branch

## Summary (required)
- Add missing migration to bring develop branch up to date  with production
- The changes for this migration are only [syntax changes to `models.py`](https://github.com/fecgov/fec-cms/commit/9513bb910b855a6ad43a8f72b76e504ca81a5ecc) that were committed to fix flake8 errors, but were missing the associated migrations

**Migration 0136 on production:**
<img width="1218" alt="Screenshot 2024-08-07 at 9 25 46 PM" src="https://github.com/user-attachments/assets/ff9a8737-a9c7-44cb-90c2-7c6397d0495a">


### Required reviewers

one dev


## Screenshots

(Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface change, include both before and after screenshots)

## How to test

- checkout develop and -run `./manage.py makemigrations --dry-run --verbosity 3`
- Should see a migration 
- checkout this branch and run  `./manage.py makemigrations --dry-run --verbosity 3`
- It should return: "no changes detected"
- `./manage.py migrate`
